vshard/storage/init.lua

```diff
@@ -260,6 +261,20 @@ else
     end
 end
 
+local function handle_results(status, ...)
```

Comment:
Nit: that's add a comment, that this is copy-pasta from `net_box.lua`, so that we don't forget that in the future